tests/cephadm/test_add_host_to_maintenance_using_flags.py

```diff
@@ -0,0 +1,41 @@
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.core_workflows import RadosOrchestrator
+from ceph.utils import get_node_by_id
+from utility.log import Log
+
+log = Log(__name__)
+
+
+def run(ceph_cluster, **kw):
+    """
+    Create a pool for rbd application that has
+    """
+    config = kw["config"]
+    cephadm = CephAdmin(cluster=ceph_cluster, **config)
+    print(cephadm)
+    orch_obj = RadosOrchestrator(node=cephadm)
+    pool_config = config.get("pool_config")
+    instance = config.get("node")
+    instance_name = get_node_by_id(ceph_cluster, instance).hostname
+
+    # Create a pool
+    out = orch_obj.create_pool(**pool_config)
+    if not out:
+        log.error("Failed to create pool")
+
+    # Put the host into maintainence
+    cmd = f"ceph orch host maintenance enter {instance_name} --force --yes-i-really-mean-it"
+    out = cephadm.shell([cmd])
+    if "moved to maintenance mode" not in str(out):
+        log.error(f"Failed to put host {instance_name} into maintenance")
+
+    # Validate Cephadm logs for no OrchestratorError
+    cmd = "ceph log last 3 cephadm"
+    out = cephadm.shell([cmd])
+    print(type(out))
+    if (
+        f"maintenance mode request for {instance_name} has SET the noout group"
+        not in str(out)
+    ):
+        log.error("Cephadm log validation failed")
```

Comment:
please add return 1 if the operation failed or raise an exception here

Reply:
Done